sycl/test/basic_tests/reqd_work_group_size.cpp

```diff
@@ -0,0 +1,109 @@
+// XFAIL: cuda || opencl
```

Comment:
There were changes in opencl specific code, so I suggest validate opencl back-end as well.

I think it better to use unittest framework with mocking API.